appveyor.yml

```diff
@@ -55,31 +50,6 @@ test_script:
   - "cd .."
   - "%CONFIGURATION%\\flattests.exe"
   - "cd tests"
-  - rem "---------------- Java -----------------"
```

Comment:
Why are we deleting these? they run on a different CI service, on a different OS (Windows). These languages should be mostly OS agnostic, but it could catch something? Are we confident this is 100% superfluous now?

Reply:
My thoughts:
1) Ideally, we'd run everything on Appveyor, too.
2) But, if we keep these Windows tests, Appveyor CI times will continue to stay very long (60 minutes wall clock time). Maybe we could try to parallelize our Appveyor builds?
3) If a port is using Windows-specific features, then maybe we should keep the port test in Appveyor.
4) If a port is not using Windows features, it seems justified (to me) that we can move its testing entirely over to Linux -- we don't need to test that the language itself runs on Windows like it does on Linux.

The least-risky thing to do here is to keep the Appveyor tests as-is. @aardappel WDYT?

Reply:
What is that 60 minutes due to? If its due to Python with Numpy, maybe just disable those? I bet the tests that rely purely on pre-installed runtimes are fast?

I'm pretty sure no Windows specific features are being used. Coverage of testsin different environments seems like a good idea to me, even in theory they're identical. And who knows at some point docker starts acting up again, then its nice to still have these tests.

I'd keep them, though we could trim them if one is particularly slow.